integrations/stripe/subscriptions/server/node/templates/invoice-payment-failed/wallet/allowance/content.txt

```diff
@@ -0,0 +1,14 @@
+Unfortunately we were unable to charge you {{amount}} {{token}}, 
+due to your wallet ({{chain}}:{{address}}) having an insufficient spending allowance.
+
+We will attempt to charge your wallet again {{next_attempt_at}}, and in order not to lose your subscription, 
+please increase your spending allowance using the Diagonal subscriptions dashboard:
+https://subscriptions.diagonal.finance/
```

Comment:
nit: allowance.diagonal.finance

Also, instead of subscriptions dashboard, please mention allowance dashboard